### DISCUSSION TITLE
tutorials/platform/consoles

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#28] [User-contributed notes]
# tutorials/platform/consoles

Console porting process: In order to develop for consoles in Godot, you need access to the console SDK and export templates for it. These export templates need to be developed either by yourself or...

https://docs.godotengine.org/en/latest/tutorials/platform/consoles.html

**Comment by FlooferLand** [comment]
I really wish there was official homebrew support for this engine, export templates for abandoned consoles like the 3Ds and the Wii U. Currently the only engine that can do this is Unity.
There is [Homebrodot](https://github.com/Homebrodot/platform-wiiu) but its not very actively maintained and it seems quite early in terms of development.

**Reply by MangoDragonHub** [reply]
That's not a bad idea at all. I think consoles like the PS3 and Wii U has sOpenGL 3 support so on the graphics back end it would compatible.

**Comment by Gnumaru** [comment]
The w4games ports should also be added to the list.

And it also should be mentioned that RAWRLAB provides a free switch port (obviously available only through the nintendo developer portal and only to licensed nintendo developers).

**Comment by karenmcd** [comment]
https://github.com/OpenOrbis/OpenOrbis-PS4-Toolchain what about this open source tool for JB units? would it be feasible and/or practical to implement this with godot? forgive my ignorance, i have no clue.
